Adding adequate CLI examples for creating triggers (#6002)
influxdata/docs-v2: content/shared/influxdb3-cli/create/token.md

The `influxdb3 create token` command creates a new authentication token. This returns the raw token string. Use it to authenticate future CLI commands and API requests.

> >[!Important]
> >InfluxDB displays the raw token string only once. Be sure to copy and securely store it.

Reviewer: ```suggestion
> [!Important]
> InfluxDB displays the raw token string only once. Be sure to copy and securely store it.
```

<!--pytest.mark.skip-->

```bash

> influxdb3 create token <COMMAND>

Reviewer: ```suggestion
influxdb3 create token <SUBCOMMAND>
```

```

## Options

| Option |          | Description            |
| :----- | :------- | :--------------------- |

> |        |`-- admin`| Create an admin token  |

Reviewer: ```suggestion
|        |`--admin`| Create an admin token  |
```

## Options

| Option |          | Description            |
| :----- | :------- | :--------------------- |
|        |`--admin`| Create an admin token  |
| `-h`   | `--help` | Print help information |

## Examples

### Create an admin token

<!--pytest.mark.skip-->

```bash
influxdb3 create token --admin
```

This returns a token string. You can use it to authenticate future requests by setting it with `--token` or the `INFLUXDB3_AUTH_TOKEN` environment variable.

### Use the token to create a database

{{% code-placeholders "YOUR_ADMIN_TOKEN|DATABASE_NAME" %}}

<!--pytest.mark.skip-->

```bash
influxdb3 create database \
  --token YOUR_ADMIN_TOKEN \
  DATABASE_NAME
```
{{% /code-placeholders %}}

Replace the following:

> - {{% code-placeholder-key %}}`YOUR_ADMIN_TOKEN`{{% /code-placeholder-key %}}: Your InfluxDB admin token

Reviewer: ```suggestion
- {{% code-placeholder-key %}}`ADMIN_TOKEN`{{% /code-placeholder-key %}}: Your InfluxDB admin token
```

## Options

| Option |          | Description            |
| :----- | :------- | :--------------------- |
|        |`--admin`| Create an admin token  |
| `-h`   | `--help` | Print help information |

## Examples

### Create an admin token

<!--pytest.mark.skip-->

```bash
influxdb3 create token --admin
```

> The output is the raw token string you can use it to authenticate future CLI commands and API requests.

Reviewer: ```suggestion
The output is the raw token string you can use to authenticate future CLI commands and API requests.
```

For CLI commands, use the `--token` option or the `INFLUXDB3_AUTH_TOKEN` environment variable to pass the token string.

### Use the token to create a database

{{% code-placeholders "YOUR_ADMIN_TOKEN|DATABASE_NAME" %}}

<!--pytest.mark.skip-->

```bash
influxdb3 create database \
  --token ADMIN_TOKEN \
  DATABASE_NAME
```
{{% /code-placeholders %}}

Replace the following:

- {{% code-placeholder-key %}}`ADMIN_TOKEN`{{% /code-placeholder-key %}}: Your InfluxDB admin token
- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: Name for your new database

> > [!Note] 
> > Set the token as an environment variable to simplify repeated commands:
> > ```bash 
> > export INFLUXDB3_AUTH_TOKEN=YOUR_ADMIN_TOKEN
> > ```

Reviewer: ```suggestion
> [!Note] 
> Set the token as an environment variable to simplify repeated CLI commands:
>
> ```bash 
> export INFLUXDB3_AUTH_TOKEN=YOUR_ADMIN_TOKEN
> ```
```
